src/Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs

```diff
@@ -93,6 +93,50 @@ public class MatchInfo
         /// <value>The text of the matching line.</value>
         public string Line { get; set; } = string.Empty;
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the matched portion of the string is highlighted.
+        /// </summary>
+        /// <value>Whether the matched portion of the string is highlighted with asterisks.</value>
+        public bool Emphasize { get; set; }
+
+        /// <summary>
+        /// Stores the starting index of each match within the line.
+        /// </summary>
+        private readonly List<int> matchIndexes;
+
+        /// <summary>
+        /// Stores the length of each match within the line.
+        /// </summary>
+        private readonly List<int> matchLengths;
+
+        /// <summary>
+        /// Stores a values indicating whether or not the terminal supports VT.
+        /// </summary>
+        private readonly bool? supportsVirtualTerminal;
```

Comment:
```suggestion
        private readonly bool? _supportsVirtualTerminal;
```

Reply:
Rather than make this field nullable, it should be `false` instead of `null` in all instances I think